### PR TITLE
chore(changelog): add missing changelogs

### PR DESCRIPTION
**Pull request by nan-wang**
chore(changelog): add missing changelogs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6489,6 +6490,22 @@ Jina is released on every Friday evening. The PyPi package and Docker Image will
...
  - [[```96e28fe8```](https://github.com/jina-ai/jina/commit/96e28fe88c57c9e586659f9916ab4b779745f10c)] __-__ __version__: the next version will be 0.9.3 (*Jina Dev Bot*)
 
+<a name=release-note-0-9-4></a>
+## Release Note (`0.9.4`)
+
+> Release time: 2021-01-11 11:17:25
+
+
+
+🙇 We'd like to thank all contributors for this new release! In particular,
```
Comment (cristianmtr) [inline CHANGELOG.md:6500]: Just `Jina Dev Bot`?

**Review by cristianmtr — CHANGES_REQUESTED** [review]
This doesn't look done

File: CHANGELOG.md [hunk 2]
```diff
@@ -6489,6 +6490,22 @@ Jina is released on every Friday evening. The PyPi package and Docker Image will
...
+ Jina Dev Bot,  🙇
+
+
+### 🍹 Other Improvements
+
+ - [[```3b62cd8b```](https://github.com/jina-ai/jina/commit/3b62cd8b4f94696fe435a7c983ff401542b748f7)] __-__ __contributor__: update contributors (*Jina Dev Bot*)
+ - [[```4b97cf3d```](https://github.com/jina-ai/jina/commit/4b97cf3df352ce869be6079b85ff0e2df1b66ab6)] __-__ __version__: the next version will be 0.9.4 (*Jina Dev Bot*)
```
Comment (hanxiao) [inline CHANGELOG.md:6507]:
```suggestion
 - [[```4b97cf3d```](https://github.com/jina-ai/jina/commit/4b97cf3df352ce869be6079b85ff0e2df1b66ab6)] __-__ __version__: the next version will be 0.9.4 (*Jina Dev Bot*)
```

This line shows this changelog is wrong. 0.9.4 release note should not show "the next version will be 0.9.4 "

**Comment by hanxiao** [comment]
@nan-wang the correct way is
```bash
./node_modules/.bin/git-release-notes v0.9.4..v0.9.5 .github/release-template.ejs > ./CHANGELOG.tmp
```

closing this PR and I will make a new one

**Comment by hanxiao** [comment]
added here: https://github.com/jina-ai/jina/blob/master/CHANGELOG.md#release-note-094

**Pull request closed without merging**